Sources/SnapshotTesting/Strategy/Any.swift

```diff
@@ -26,8 +26,10 @@ private func snap<T>(_ value: T, name: String? = nil, indent: Int = 0) -> String
     let subjectType = String(describing: mirror.subjectType)
       .replacingOccurrences(of: " #\\d+", with: "", options: .regularExpression)
     description = count == 0 ? "\(subjectType).none" : "\(subjectType)"
-  case (let value as SnapshotStringConvertible, _):
+  case (let value as AnySnapshotStringConvertible, _) where type(of: value).renderChildren:
```

Comment:
This also contains a fix for #65 where we can opt into omitting child mirror nodes.